mosaic-runtime/src/commonMain/kotlin/com/jakewharton/mosaic/ui/Color.kt

```diff
@@ -110,6 +116,9 @@ public value class Color internal constructor(
 	}
 }
 
+@PublishedApi
+internal const val UnspecifiedColor: Int = Int.MIN_VALUE
```

Comment:
Are we just assuming no one is going to use 0x80000000? Or do we basically ignore alpha channel? I don't recall after the switch to true color seeing whether it was supported or not.

Reply:
Yes, I specifically chose a well-known constant for simplicity, and yes, now we ignore the alpha channel, since, at least, I did not see the possibility of using it in the terminal. If we want an alpha channel, then we just need to change the field in Color to long, because otherwise 32 bits will not be enough for such a constant.

Reply:
The only way we'd support alpha is to premultiply it. So if you had a background of red and chose yellow at 50% we'd do the math and turn the text opaque orange. Not with the bother at this time.